user_tools/docs/index.md

```diff
@@ -55,57 +55,62 @@ The wrapper has the following deployment modes:
++==========+===============+=============+=================================+==========+
+| EMR      | qualification | local       | spark_rapids_user_tools \       |  23.02+  |
+|          |               |             |   emr qualification [ARGS]      |          |
+|          |               +-------------+---------------------------------+----------+
+|          |               | serverless  | spark_rapids_user_tools \       |  23.02+  |
+|          |               |             |   emr qualification [ARGS] \    |          |
+|          |               |             |   --mode=serverless             |          |
+|          |               +-------------+---------------------------------+----------+
+|          |               | cluster     | unsupported                     |    N/A   |
+|          +---------------+-------------+---------------------------------+----------+
+|          | profiling     | local       | unsupported                     |    N/A   |
+|          |               +-------------+---------------------------------+----------+
+|          |               | serverless  | unsupported                     |    N/A   |
+|          |               +-------------+---------------------------------+----------+
+|          |               | cluster     | unsupported                     |    N/A   |
+|          +---------------+-------------+---------------------------------+----------+
+|          | bootstrap     | local       | spark_rapids_user_tools \       |  23.02+  |
+|          |               |             |   emr bootstrap [ARGS]          |          |
+|          |               +-------------+---------------------------------+----------+
+|          |               | serverless  | unsupported                     |    N/A   |
+|          |               +-------------+---------------------------------+----------+
+|          |               | cluster     | spark_rapids_user_tools \       |  23.02+  |
+|          |               |             |   emr bootstrap [ARGS]          |          |
++----------+---------------+-------------+---------------------------------+----------+
+| Dataproc | qualification | local       | spark_rapids_user_tools \       |  23.04+  |
+|          |               |             |   dataproc qualification [ARGS] |          |
+|          |               +-------------+---------------------------------+----------+
+|          |               | serverless  | unsupported                     |    N/A   |
+|          |               +-------------+---------------------------------+----------+
+|          |               | cluster     | spark_rapids_dataproc \         | 22.10.1+ |
+|          |               |             |   qualification [ARGS]          |          |
+|          +---------------+-------------+---------------------------------+----------+
+|          | profiling     | local       | spark_rapids_user_tools \       |  23.04+  |
```

Comment:
Let's not add the local profiling and bootstrap support officially until that's going to be in a PR as we may release minor user tools version before those are ready.

Reply:
Done!
I put the major upcoming release because I was not sure about the micro version though.